VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -129,9 +124,6 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         final FrameLayout nameLayout = (FrameLayout) layout.findViewById(R.id.name_layout);
         imageView = (ImageView) layout.findViewById(R.id.profile_image);
         if (config.isUserProfilesEnabled()) {
-            if (null != profileImage) {
-                loadProfileImage(profileImage, imageView);
-            }
```

Comment:
This was actually useful, since it handled the case were the data was returned before view creation (e.g. in the case of delayed Fragment attachment, or view refresh).